Superposition/Superposition.ipynb

```diff
@@ -367,37 +367,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 11*. Hardy state.\n",
```

Comment:
Please add the new task to the Notebook as well, so that everybody who attempts to solve the kata can enjoy it regardless of the format they use :-)